### PR TITLE
Added first draft of new section: 'Updating Service Registry artifacts', and minor edits of the Downloading section 

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

**Comment by bredamc** [comment]
@wtrocki @bibryam This PR adds a section about updating Service Registry artifacts. Please review for technical accuracy and let me know if you approve or have any more updates. Thank you!

File: rhoas-cli-service-registry/README.adoc [hunk 1]
```diff
@@ -251,7 +263,38 @@ The artifact is an Apache Kafka Avro schema in JSON format.
 [source,shell]
 ----
 $ wget https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
+--2021-09-27 16:17:18--  https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
+Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...
```
Comment (wtrocki) [inline rhoas-cli-service-registry/README.adoc:267]: I think that particular output is very verbose and doesn't give any value to end user. wget is also linux specific command. I think we can replace all of that with one sentence - download example file

**Review by wtrocki — APPROVED** [review]
Looks good.

File: rhoas-cli-service-registry/README.adoc [hunk 2]
```diff
@@ -327,35 +375,119 @@ For more information about any of the options, view the command help `rhoas serv
...
 --
 
+[NOTE]
+====
+You can use additional options, such as `--group` and `--instance-id`, to specify the artifact to download.
+For more information about any of the options, view the command help `rhoas service-registry artifact download -h` and `rhoas service-registry artifact get -h`.
+====
+
+[id="proc-updating-service-registry-artifacts_{context}"]
+== Updating Service Registry artifacts
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply rhoas-cli-service-registry/README.adoc:423]: I think this is outside scope of this PR but we really need to discuss it as follow up.

CC @bhardesty

File: rhoas-cli-service-registry/README.adoc [hunk 3]
```diff
@@ -327,35 +375,119 @@ For more information about any of the options, view the command help `rhoas serv
...
+* Use the `rhoas service-registry artifact get` command and specify the artifact ID.
+* Use the `rhoas service-registry artifact download` command and specify the global ID.
+
+[NOTE]
+====
+To find the artifact ID, use the `rhoas service-registry artifact list` command.
+
+To find the global ID, see the Kafka Message payload.
+====
+
```
Comment (smccarthy-ie) [inline rhoas-cli-service-registry/README.adoc:397]: Suggest "Kafka message payload" (no need for capital in Message)

**Review by smccarthy-ie — APPROVED** [review]
LGTM, thanks!

**Review by smccarthy-ie — APPROVED** [review]

**Review by wtrocki — APPROVED** [review]

**Pull request merged**